AntonioLok/online-shopping-reactJS-Frontend
# remove backend logic

# Changelog
- Remove all back-end related code from this repo

# Test
- Install dependencies using `npm i`
- Start web app using `npm run start-webapp`
- App should now be running on port 8080

# To do after
- Login component on the front end
